docs/team/candyhy.md

```diff
@@ -9,3 +9,61 @@ PocketEstate enables easy organisation of mass clientele property information th
+
+<div style="page-break-after: always;"></div>
+
+
+* **Enhancement**: Added the ability to list properties and appointments through the three different `list` commands - `list all`, `list property`, `list appointment`.
+    * What it does: Allows the user to list a property, list an appointment or list both properties and appointments to the application.
+    * Justification: These basic core commands are required for users to view information in the application.
+    * Highlights: Allows users to retrieve appointments or properties, separately or together after a sort or filter command.
+
+
+* **Additional Enhancements**:
+    * Wrote additional tests ([\#324](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/324), [\#204](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/204), [\#205](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/205), [\#300](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/300), [\#306](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/306), [\#329](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/329), [\#335](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/335))
+
+#### Documentation
+
+* **User Guide**:
+    * Added documentation for commands `list all`, `list property` and `list appointment` ([\#39](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/39))
+    * Added User Interface section ([\#343](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/343))
+    * Annotate Ui diagrams ([\#339](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/339))
+
+<div style="page-break-after: always;"></div>
+
+* **Developer Guide**:
+    * Added effort section of DG. ([\#322](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/322))
+    * Added user stories and use cases for list command. ([\#314](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/314), [\#38](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/38))
+    * Updated Storage component and Storage class diagram ([\#199](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/199))
+    * Formatted Appendix with numbering for each section ([\#322](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/322))
+
+#### Community
+
+* PRs reviewed (with non-trivial review comments): ([\#162](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/162), [\#126](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/126), [\#298](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/298), [\#111](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/111))
+* Provided tips through forum discussions (examples: [Configuring IntelliJ to follow module coding standards for switch statements](https://github.com/nus-cs2103-AY2021S2/forum/issues/42))
+* Provided help through forum discussions (examples: [\#23](https://github.com/nus-cs2103-AY2021S2/forum/issues/23))
+* Fixed a bug in the original AB3 Storage class diagram (issue: [\#273](https://github.com/nus-cs2103-AY2021S2/forum/issues/273))
```

Comment:
I think need to rephrase this because you raised the bug but didn't fix it right?

Reply:
oh right ok, will say I reported a bug instead